lib/queries/useAuthoritiesBrowse/useAuthoritiesBrowse.js

```diff
@@ -143,14 +143,6 @@ const useAuthoritiesBrowse = ({
   const [currentIndex, setCurrentIndex] = useState(searchIndex);
   const [offset, setOffset] = useState(0);
 
-  useEffect(() => {
```

Comment:
In MARC Authority app - will page reset when query, search index or filters change?

Reply:
Yes, page is reset in `useBrowserPaging` in:

```
useEffect(() => {
    updatePage(0, _initialQuery);
    setMainRequestSearch(getMainRequestSearch(_initialQuery, 0));
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [_initialQuery, initialIndex]);
```

Reply:
The page doesn't reset after changing the filter. This behavior was previously.